src/DependencyInjection/Configuration.php

```diff
@@ -31,6 +31,15 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->children()
                 ->booleanNode('cache_sqlite_db')->defaultFalse()->end()
+                ->arrayNode('cache_db')
```

Comment:
should we then remove `cache_sqlite_db` ?

Reply:
Yes, I'll remove it.
https://github.com/liip/LiipFunctionalTestBundle/pull/398#issuecomment-365875327